compose/foundation/foundation/build.gradle

```diff
@@ -261,3 +262,12 @@ androidx {
     description = "Higher level abstractions of the Compose UI primitives. This library is design system agnostic, providing the high-level building blocks for both application and design-system developers"
     legacyDisableKotlinStrictApiMode = true
 }
+
+tasks.register("updateEmojiValues", UpdateEmojiValuesTask.class) {
```

Comment:
I suggest to create a task  `updateInternalization` that depends on `updateEmojiValues` and `updateTranslations`, and describe it in `MULTIPLATFORM.md`.

Even if these tasks are needed to be called:
- after updating Unicode version
- after merging Jetpack Material or after adding a language

It is better to have one task that we call every merge, every unicode version update, etc. It makes life simpler (with an option of calling these tasks separately if needed)

Reply:
Good idea, but maybe not "Internationalization" as it's not really about that.
Maybe "runAllUpdateTasks"?

Reply:
I would exclude `run` and `task` as it usually implied when we use Gradle.

Maybe `updateGeneratedCode`? And have:
- updateGeneratedEmoji
- updateGeneratedTranslations

Reply:
`updateGeneratedCode` is better, but what if we have more tasks like these (in the sense that we need to run them whenever we do something, e.g. merge), but they don't actually generate code?

Reply:
>  but they don't actually generate code?

besides code generation I can only think about these tasks we might need to run after a merge:
- run tests
- check/dump API

I probably wouldn't unite them with code generation.

Reply:
not relevant with ICU